app/admin/user.rb

```diff
@@ -167,6 +167,18 @@
         end
       end
 
+      tab I18n.t('experience') do
+        panel I18n.t('user_open_source') do
+          table_for user.contributions.valid_pull_requests.order(created_at: :desc), i18n: Contribution, id: 'table_admin_contributions' do
+            column :name, i18n: Repository do |contribution|
+              contribution.repository.decorate.name
```

Comment:
here would be nice to decorate the contribution and, in the decorator add the decorate for the relation

Reply:
Done